Website/src/rendering/src/components/Products/ProductDetailPage.tsx

```diff
@@ -0,0 +1,340 @@
+import { useRouter } from 'next/router';
+import { Spec, Variant } from 'ordercloud-javascript-sdk';
+import { FormEvent, useCallback, useEffect, useState } from 'react';
+import useOcProductDetail from 'src/hooks/useOcProductDetail';
+import { createLineItem } from 'src/redux/ocCurrentCart';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import { logViewEvent } from 'src/services/CdpService';
+import ProductQuantityInput from './ProductQuantityInput';
+import ProductSpecField from './ProductSpecField';
+import { Product } from './Shop';
+
+type ProductProps = {
```

Comment:
`props` in react is a term to describe the input properties of a component. Here, I believe this type is not used as the input of a component. `Props` could be removed from its name.